spec/controllers/api/v1/reviews_controller_spec.rb

```diff
@@ -27,6 +27,15 @@
     )
   }
 
+  let!(:second_review) {
+    Review.new(
+      rating: 4,
+      body: "Hi world",
+      park: first_park,
+      user: first_user
+    )
+  }
+
```

Comment:
This is taking up an enormous amount of space. I would highly suggest implementing factorybot, for time and DRYness, but wont require it for this PR